Marlin/src/inc/Conditionals_post.h

```diff
@@ -1485,13 +1485,14 @@
   /**
    * Delta radius/rod trimmers/angle trimmers
    */
-  #define _PROBE_RADIUS (DELTA_PRINTABLE_RADIUS - (MIN_PROBE_EDGE))
+  #if HAS_BED_PROBE
+    #define _PROBE_RADIUS (DELTA_PRINTABLE_RADIUS - _MAX(HYPOT(probe_offset.x, probe_offset.y), ABS(MIN_PROBE_EDGE)))
```

Comment:
Users are meant to understand that they must set a reasonable `MIN_PROBE_EDGE` value. It is always a little sketchy to allude to specific implementation and variables in these macros, and we prefer to avoid it. It would be better to move the definition of `_PROBE_RADIUS` and `DELTA_CALIBRATION_RADIUS` out of this file and define them as functions or methods in `delta.h`/`delta.cpp` instead.